### PR TITLE
[New Exercise]: Clock

### PR DESCRIPTION
**Pull request by glaxxie**

### CODE REVIEW

File: exercises/practice/clock/Clock.ps1 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Note: Please try to implement the class and its method instead of using built-in module Datetime.
+
+.EXAMPLE
+    $clock1 = [Clock]::new(5,0)
+    $clock1.ToString()  => "05:00"
+
+    $clock2 = [Clock]::new(6,-120)
+    $clock2.Add(60).ToString() => "05:00"
+
+    $clock1 -eq $clock2 => True
```
Comment (meatball133) [inline exercises/practice/clock/Clock.ps1]:
```suggestion
    $clock1 -eq $clock2 => $True
```

File: exercises/practice/clock/Clock.ps1 [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+.DESCRIPTION
+    Implement a clock that handles times without dates in 24 hours format.
+    You should be able to add and subtract minutes to it.
+    Two clocks that represent the same time should be equal to each other.
+    Note: Please try to implement the class and its method instead of using built-in module Datetime.
+
+.EXAMPLE
+    $clock1 = [Clock]::new(5,0)
+    $clock1.ToString()  => "05:00"
```
Comment (meatball133) [inline exercises/practice/clock/Clock.ps1]:
```suggestion
    $clock1.ToString() => "05:00"
```

File: exercises/practice/clock/.meta/Clock.example.ps1 [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    Note: Please try to implement the class and its method instead of using built-in module Datetime.
+
+.EXAMPLE
+    $clock1 = [Clock]::new(5,0)
+    $clock1.ToString()  => "05:00"
+
+    $clock2 = [Clock]::new(6,-120)
+    $clock2.Add(60).ToString() => "05:00"
+
+    $clock1 -eq $clock2 => True
```
Comment (meatball133) [inline exercises/practice/clock/.meta/Clock.example.ps1]:
```suggestion
    $clock1 -eq $clock2 => $True
```

File: exercises/practice/clock/.meta/Clock.example.ps1 [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+.DESCRIPTION
+    Implement a clock that handles times without dates in 24 hours format.
+    You should be able to add and subtract minutes to it.
+    Two clocks that represent the same time should be equal to each other.
+    Note: Please try to implement the class and its method instead of using built-in module Datetime.
+
+.EXAMPLE
+    $clock1 = [Clock]::new(5,0)
+    $clock1.ToString()  => "05:00"
```
Comment (meatball133) [inline exercises/practice/clock/.meta/Clock.example.ps1]:
```suggestion
    $clock1.ToString() => "05:00"
```

**Review by meatball133 — COMMENTED** [review]
Some tiny nits

File: exercises/practice/clock/Clock.ps1 [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    $clock2 = [Clock]::new(6,-120)
+    $clock2.Add(60).ToString() => "05:00"
```
Comment (meatball133) [inline exercises/practice/clock/Clock.ps1]: I think it could make sense to use `Return` as the other 2 prs instead of `=>`.

Comment (glaxxie) [reply exercises/practice/clock/Clock.ps1]: All 3 PRs fixed!

**Review by meatball133 — APPROVED** [review]

**Pull request merged**